.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+---
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: monthly
+      time: "10:00"
+    open-pull-requests-limit: 99
+    commit-message:
+      prefix: "CHORE: "
+    reviewers:
+      - kimchelly
```

Comment:
Can we do the evergreen team? Should this only be you?

Reply:
I talked with Brian about this and unfortunately, the diffusion of responsibility of assigning multiple people will probably make it so that only a subset of people (or possibly nobody) would become responsible for merging these. Since PLT is the only team with so many repos, I imagine we would be the only ones looking at the non-Evergreen Dependabot PRs anyways.